topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to recurrent neural networks (RNN)
+zenodo_link: https://zenodo.org/record/4477881
+questions:
+- What is a recurrent neural network (RNN)?
+- What are some applications of RNN?
+objectives:
+- Understand the difference between feedforward neural networks (FNN) and RNN
+- Learn various RNN types and architectures
+- Solve a sentiment analysis problem on IMDB movie review dataset using RNN in Galaxy
```

Comment:
Maybe add one more objective: `Learn how to create a neural network using Galaxy's deep learning tools`

Reply:
Added this.